bokehjs/src/coffee/models/transforms/interpolator.coffee

```diff
@@ -0,0 +1,85 @@
+_ = require "underscore"
+Transform = require "./transform"
+p = require "../../core/properties"
+
+class Interpolator extends Transform.Model
+
+  initialize: (attrs, options) ->
+    super(attrs, options)
+    @_x_sorted = []
+    @_y_sorted = []
+    @_sorted_dirty = true
+
+    @bind 'change', () ->
+      console.log 'Changed the configuration'
```

Comment:
I will remove this as it was not meant to stay and was just some debugging stuff for me.